app/models/form_question.rb

```diff
@@ -9,4 +9,21 @@ def self.get_form_question_types
   def self.get_questions_for_form(form_type)
   	Form.where(form_name: form_type)[0].form_questions.sort_by {|e| e.order}
   end
+
+  def self.get_list_of_questions(form_type)
```

Comment:
Everything is passed as form_type (form_name) from the user controller and user view (show.html.haml)